app/views/hyrax/base/_form_member_of_collections.html.erb

```diff
@@ -0,0 +1,46 @@
+<%# OVERRIDE Hyrax 2.6.0 to include bulkrax collections in the dropdown in the work/edit form relationships tab %>
```

Comment:
Can you add a comment where the code is different from Hyrax?

Reply:
yes of course - done! @labradford